src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -528,11 +533,17 @@ protected function push($identifier, $options) {
       $this->say("Pushing artifact to git.remotes...");
     }
 
+    $arguments = "";
+
+    if ($options['truncate-history']) {
```

Comment:
@nedsbeds I think this needs `if ($options['truncate-history'] || $this->truncateHistory) {` otherwise it will ignore the blt settings in yml on deploy.

Reply:
Ok that should be fixed now :)